content/general/concepts/software-development-life-cycle/terms/rapid-application-development/rapid-application-development.md

```diff
@@ -0,0 +1,42 @@
+ ---
+Title: 'Rapid Application Development (RAD)'
+Description: 'Rapid Application Development (RAD) is a software development methodology that emphasizes quick prototyping and iterative development to meet the requirements of a project.'  
+
+Subjects:
+  'Mobile Development'
+  'Web Development'  
+Tags:
+  'Documentation'
+  'Best Practices'
+  'Development'  
+CatalogContent:
+  'introduction-to-open-source'
+  'paths/computer-science'
+---
+**Rapid Application Development (RAD)** is a software development methodology that emphasizes quick prototyping and iterative development to meet the requirements of a project. The primary goal of RAD is to reduce development time and costs by quickly delivering a working prototype to the client for feedback and iteration.
```

Comment:
I believe that iterative prototyping and close collaboration with stakeholders is the primary goal of RAD. Perhaps we can include "reduce development time and costs by quickly delivering a working prototype to the client for feedback and iteration" as the ideal result of using this methodology